<launch>
	<!--include file="$(find usb_cam)/launch/usb_cam-test.launch"-->

	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
		<param name="video_device" value="/dev/video1" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
		<param name="pixel_format" value="yuyv" />
		<param name="camera_frame_id" value="usb_cam" />
		<param name="io_method" value="mmap"/>
		<param name="framerate" value="40"/>
	</node>

	<node pkg="lane_detector" type="lane_detector" name="lane_detector">
		<param name="resize_width" value="480" />
		<param name="resize_height" value="270" />
		<param name="steer_max_angle" value="45" />

		<param name="bin_thres" value="110" />	
		<param name="detect_y_offset" value="30" />	
		<!--  control_value *= control_factor / 100  -->
		<param name="control_factor" value="100" />
		<param name="throttle" value="1515" />
		<remap from="/lane_detector/write" to="write" />
	</node>

</launch>
